test/algorithms/set_operations/difference/difference.cpp

```diff
@@ -34,15 +34,6 @@
 namespace
 {
 
-// Change compiler defines to constexpr bools
-// to make conditions more readable
-// and to always compile all code.
```

Comment:
This was only done, once now, in this source, so I harmonized it back with the others.